src/python/pants/engine/session.py

```diff
@@ -0,0 +1,19 @@
+# Copyright 2016 Pants project contributors (see CONTRIBUTORS.md).
```

Comment:
Should this be in `engine/internal`? The question is mostly if we expect to document the type `SesssionValues` and we expect plugin authors to directly import the value. Generally, this gets abstracted from them fwict.

Reply:
It might depend what API we end up with for `RuleRunner`/`TestBase`...? I guess it's unlikely that the actual `SessionValues` type would be exposed in that API though.

Reply:
> I guess it's unlikely that the actual SessionValues type would be exposed in that API though.

Yeah, I suspect it's too low level. We can also always "promote" the API to be public if we realize it's relevant. It's harder to demote it to private.